src/TypeCollector.php

```
@@ -223,18 +271,26 @@ private function collectProperties(): array
+                        if ($type->allowsNull() && $typeName !== 'null') {
+                            $properties[$property->getName()] = [$typeName, 'null'];
+                        } else {
+                            $properties[$property->getName()] = $typeName;
+                        }
                     }
                 } else {
                     $properties[$property->getName()] = '';
                 }
 
-                foreach ($property->getAttributes(Attribute\Timestamp::class) as $ignored) {
+                foreach ($property->getAttributes(Timestamp::class) as $ignored) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$ignored'. (undefined)

(UnusedLocalVariable)